### PR TITLE
gallery page improved

### PR DESCRIPTION
**Pull request by bishalx0**
- If it works fine, we can extend the page.

### CODE REVIEW

**Comment by bishalx0** [comment]
we can link completed events from the event page to their respective gallery section, think about it 🤔

**Review by Abhilekhgautam — CHANGES_REQUESTED** [review]
1. Can we do something with this gap?
image

Everything else looks good to me

File: site/html/gallery.html [hunk 1]
```diff
@@ -25,181 +30,145 @@
...
-          <img src="/site/images/slide5.jpg" alt="" class="galleryitems mb-4">
-        </div>
-        <div class="col">
-          <img src="/site/images/slide5.jpg" alt="" class="galleryitems mb-4">
-        </div>
-        <div class="col">
-          <img src="/site/images/slide5.jpg" alt="" class="galleryitems mb-4">
+      <section class="sec-container" id="SP_linkedlist">
+        <div class="heading">
+          <h3>Photos from Linked List Implementation in Java</h3>
```
Comment (Abhilekhgautam) [inline site/html/gallery.html:120]: Event headings to h2?

Comment (bishalx0) [reply site/html/gallery.html:120]: This is not the subject of concern, we can take it as a feature. This is just due to image size; showing the original width-height ratio without size reduction makes sense.
Surely heading will change.

**Review by Abhilekhgautam — APPROVED** [review]
Everything looks ok to me.

**Pull request merged**